apps/opik-backend/src/test/resources/config-test.yml

```diff
@@ -3,6 +3,7 @@ logging:
   level: INFO
   loggers:
     com.comet: DEBUG
+    org.jdbi: DEBUG
```

Comment:
This will significantly increase the test logs. Why do we need this?

Reply:
It helped me debug why ORDER BY wasn't working in the query I improved. I thought to leave it and see what the reviewer says :)
Will remove.

Reply:
Yeah, it's better to remove and use only when necessary. IMHO